File: programmers/기초/수 조작하기 1.js
// 수 조작하기 1
// "w" : n이 1 커집니다.
// "s" : n이 1 작아집니다.
// "d" : n이 10 커집니다.
// "a" : n이 10 작아집니다.

function solution(n, control) {
  for (const i of control) {
    if (i === "w") {
      n += 1;
    }
    if (i === "s") {
      n -= 1;
    }
    if (i === "d") {
      n += 10;
    }
    if (i === "a") {
      n -= 10;
    }
  }
  return n;
}

File: programmers/python/수 조작하기 1.py
# 수 조작하기 1
def solution(n, control):
    answer = n
    c = { 'w' : 1, 's' : -1, 'd' : 10, 'a' : -10 }

    for i in control:
        answer += c[i]
        
    return answer